### DISCUSSION TITLE
[Feature] Marquee Text Control

### DISCUSSION THREAD

**Discussion by Avid29** [CommunityToolkit/Labs-Windows#231] [Proposals]
### Describe the problem

Looking for a convenient TextBlock that can scroll text like a Marquee

### Describe the solution

A MarqueeText control that uses a Storyboard and two TextBlocks to scroll text across the screen if it doesn't fit

### Additional info

I wrote a control like this for StrixMusic, however I think I could do it better if I took another go at it

### Help us help you

Yes, I'd like to be assigned to work on this item.

**Comment by ghost** [comment]
Hello, 'Avid29! Thanks for submitting a new feature request. I've automatically added a vote 👍 reaction to help get things started. Other community members can vote to help us prioritize this feature in the future!

**Comment by michael-hawker** [comment]
Thanks @Avid29, sounds like a great thing for Toolkit Labs 🧪 CommunityToolkit/WindowsCommunityToolkit#4487. We'll hopefully be in a good spot later in the month to start onboarding experiments to test our infrastructure. Can give you a ping then if you want to help be a guinea pig for us to help try it out.

**Comment by Avid29** [comment]
@michael-hawker I'd love to participate however you're offering

**Comment by Arlodotexe** [comment]
I've moved this to Labs @Avid29. We'll have more on how to set up a full proposal soon:tm: (early next week)

**Comment by michael-hawker** [comment]
Oops, we closed the original issue. @Arlodotexe we should get the templates reset up and maybe include the new Tasklist stuff we discovered as a flat list of the checklist items components need.

Just noticed we missed the default (and incorrect) namespace VS adds due to the Uno issue:

https://github.com/CommunityToolkit/Labs-Windows/blob/c55b3a8bf47f85c614086d1d6e92d853c264d7d3/components/MarqueeText/src/MarqueeText.cs#L6

Was also wondering if we should call this a `MarqueeTextBlock` as a name? Thoughts?
